src/Microsoft.Health.Dicom.Api/Features/Exceptions/ExceptionHandlingMiddleware.cs

```diff
@@ -90,8 +90,7 @@ private IActionResult MapExceptionToResult(Exception exception)
             case AuditHeaderTooLargeException:
             case ConnectionResetException:
             case OperationCanceledException:
-            case DicomImageException ex when IsTaskCanceledException(ex.InnerException):
-            case DataStoreException e when IsTaskCanceledException(e.InnerException):
+            case Exception e when IsOperationCanceledException(e.InnerException):
```

Comment:
Instead of doing for all Exception, may be limit to either `DicomServerException` or `MicrosoftHealthException`

Reply:
Talked offline; initially did this because the RBAC exception base was just "Exception". Updated to be MicrosoftHealthException so we can reduce scope here